app/views/comments/_comment_date.erb

```diff
@@ -1,7 +1,15 @@
 <div class="comment-date">
   <a href="<%= decorated_comment.path %>">
+  <span class="hidden m:inline-block">
     <time datetime="<%= decorated_comment.published_timestamp %>">
       <%= decorated_comment.readable_publish_date %>
     </time>
+    </span>
   </a>
-</div>
+  <% if !decorated_comment.edited_at? %>
+    •
+    <span title="<%= decorated_comment.edited_at.strftime('%A, %d %B %Y, %T') %>">
+      EDITED
+    </span>
+  <% end %>
+</div>
```

Comment:
Just need an extra blank line at the end of the file to make it pass checks.

Reply:
@Link2Twenty here is a better view of the file as at now. 
![comment_date](https://user-images.githubusercontent.com/56835876/95576258-0ee83280-0a28-11eb-8dd7-89f86dcff097.jpg) What needs correction?

Reply:
the blank line is only needed after the very last close div (so after line 18)

Reply:
Also at line 15 (in the image) it should look like this

```erb
    • <time datetime="<%= decorated_comment.edited_at %>">Edited</time>
```

Reply:
Noted